lms/djangoapps/teams/static/teams/js/views/teams.js

```diff
@@ -0,0 +1,26 @@
+;(function (define) {
+    'use strict';
+    define([
+        'teams/js/views/team_card',
+        'common/js/components/views/paginated_view'
+    ], function (TeamCardView, PaginatedView) {
+        var TeamsView = PaginatedView.extend({
+            type: 'teams',
+            cardView: TeamCardView,
+
+            events: {
+                'click button.action': '' // entry point for team creation
+            },
+
+            render: function () {
+                PaginatedView.prototype.render.call(this);
+
+                this.$el.append(
+                    $('<button class="action action-primary">' + gettext('Create new team') + '</button>')
```

Comment:
Perhaps not styled yet, but does this button fall under the same pattern as other "Add object" buttons in the LMS? If so, then the label should probably be "Add team"
@dan-f @marcotuts @frrrances ?